corehq/elastic.py

```diff
@@ -144,7 +179,7 @@ class ESError(Exception):
     pass
 
 
-def run_query(index_name, q, debug_host=None):
+def run_query(index_name, q, debug_host=None, es_instance=ES_DEFAULT_INSTANCE):
```

Comment:
i kind of wanted all the EXPORT_INSTANCES defined in one place in case this ever gets reused it's less likely someone creates the same exact es instance with the same params. that param name is a bit misleading though. will rename